docker-compose-development.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
```

Comment:
Yeah, removing .env is fine with me, I'll be curious to hear if anyone runs into conflicts with the chosen ports. Regardless, doesn't sound like it'll need to be something that has to be worked around for long since compose 1.9 will give us the best of both worlds.